todoapp/app.py

```diff
@@ -1,9 +1,10 @@
-from trello_request import *
+import os
```

Comment:
Moving this file means .env also needs to have `FLASK_APP=todoapp/app` but .env.template hasn't been updated.